app/cdap/components/WranglerGrid/SelectColumnPanel/ColumnsList/__test__/SelectColumnsList.test.tsx

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { fireEvent, render, screen } from '@testing-library/react';
+import history from 'services/history';
+import React from 'react';
+import { Route, Router, Switch } from 'react-router';
+import SelectColumnsList from 'components/WranglerGrid/SelectColumnPanel/ColumnsList';
+
+describe('It should test the SelectColumnsList Component', () => {
+  it('should render the SelectColumnsList Component', () => {
+    render(
+      <Router history={history}>
+        <Switch>
+          <Route>
+            <SelectColumnsList
+              selectedColumnsCount={1}
+              setSelectedColumns={() => jest.fn()}
+              dataQuality={[]}
+              transformationDataType={[]}
+              columnsList={[]}
+              transformationName={''}
+              selectedColumns={[]}
+            />
+          </Route>
+        </Switch>
+      </Router>
+    );
+    expect(screen.getByTestId(/select-column-list-parent/i)).toBeInTheDocument();
+  });
+  it('should render the SelectColumnsList Component with some input value along with label and null and trigger input ', () => {
```

Comment:
I'm not seeing much value added by these tests...

Reply:
In this test case we are checking on click of search icon the input tag style should change, the test case is updated with style assertion